podcasts/TranscriptGradientView.swift

```diff
@@ -0,0 +1,52 @@
+import UIKit
+
+class TranscriptGradientView: UIView {
```

Comment:
Nitpick Suggestion: Could this be a generic gradient view? to be used on other places?

Reply:
Done!

I also went ahead and fix a bug that wasn't updating the gradient colors when the episode changed.

Reply:
Thanks! will have a look again. Was the bug when the player went to the next episode when the previous episode finished?

Reply:
Yep